lib/brains/tests/test_client.py

```diff
@@ -22,7 +22,9 @@ def test_normal(self):
 
```

Comment:
The mocks get cleared on each set_mocks call, so you can do it multiple times in a test and do the different responses.

Clearing mock data on tear down is a good policy though.